File: Beginner/1009 - Salary with Bonus/1009.cs
using System;

class URI
{
    static void Main(string[] args)
    {
        String name = Console.ReadLine();
        double A = Double.Parse(Console.ReadLine());
        double B = Double.Parse(Console.ReadLine());

        double salario = A + (B * 0.15);

        Console.WriteLine($"TOTAL = R$ {salario:0.00}");
    }
}

File: Beginner/1009 - Salary with Bonus/1009.c
#include <stdio.h>

int main()
{
    char nome[15];
    double A, B, salario;

    scanf("%s", nome);
    scanf("%lf %lf", &A, &B);

    salario = A + (B * 0.15);
    printf("TOTAL = R$ %.2lf\n", salario);

    return 0;
}